tasks/kubedriver/add_kubedriver_deployment_location.yaml

```diff
@@ -0,0 +1,47 @@
+- name: Authenticate with LM
+  uri:
+    url: "{{ drivers_onboard_addr }}/oauth/token"
+    validate_certs: no
+    method: POST
+    body_format: form-urlencoded
+    body:
+      grant_type: "client_credentials"
+    headers:
+      authorization: "Basic {{ (drivers_onboard_client + ':' + drivers_onboard_client_secret) | b64encode }}"
+    return_content: yes
+    status_code: 200
+  register: lm_auth_response
+
+- name: Set Access Token fact
+  set_fact:
+    lm_access_token: "{{ lm_auth_response.json.access_token }}"
+
+- name: Get kubectl config via microk8s.config
+  shell: source /etc/profile.d/apps-bin-path.sh && microk8s.config view
+  args:
+    executable: /bin/bash
+  register: microk8s_config_result 
+
+- set_fact:
+    kubeconfig: "{{ microk8s_config_result.stdout }}"
+
```

Comment:
We should try to make anything new we add to these scripts idempotent (that is, it should be possible to re-run the installation and the scripts should cope e.g. if the deployment location already exists, it should skip the creation and continue). Could you add a check in here to determine if the deployment location exists and, if so, skip the create?

Reply:
Good idea, I'll add that.